cmd/loom/gateway_reactors.go

```diff
@@ -4,13 +4,25 @@ package main
 
 import (
 	"fmt"
+	"time"
 
 	glAuth "github.com/loomnetwork/go-loom/auth"
+	"github.com/loomnetwork/go-loom/client"
 	"github.com/loomnetwork/loomchain/config"
 	"github.com/loomnetwork/loomchain/fnConsensus"
+	"github.com/loomnetwork/loomchain/log"
 	tgateway "github.com/loomnetwork/transfer-gateway/gateway"
 )
 
+type ServiceName string
```

Comment:
Maybe we don't actually need a new type here. Using just const without type should be enough.